README.md

```diff
@@ -98,6 +101,20 @@ window.pluginBridge.register({
 
 The parent (Project Griffon UI) calls init once the plugin is registered. Here, Project Griffon UI will pass in any applicable configuration settings.
 
+##### customize
+
+The parent calls customize with values that change how the plugin itself should behave.
+
+Example Payload
+```
+{ theme: 'light', locale: 'en-US', showTimeline: false }
+```
+
+**Supported Values**
+- *theme*: (light/dark) specifies the theme the parent's UI is using
+- *locale*: (en-US, etc) specifies which localization to use in the UI
+- *showTimeline*: (true/false) many Griffon Plugins show a timeline at the bottom by default. Setting this to `false` will hide that timeline.
```

Comment:
These don't seem to be plugin specific and instead are global settings that are being passed down. Having the ability to have global settings and plugin specific settings seems like something we should accommodate. 

Are any of these values passed to `init` and when is this customize function expected to be called? Would be good to try to clarify the behaviors around that.

Would `settings` be a better name for the function instead of `customize`? And having the ability to distinguish between global and plugin settings seems ideal. Is there a way for the plugin to change these settings?